patches/react-native-web+0.19.9+005+fixLastSpacer.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js b/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
+index faeb323..68d740a 100644
+--- a/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
++++ b/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
+@@ -78,14 +78,6 @@ function scrollEventThrottleOrDefault(scrollEventThrottle) {
+ function windowSizeOrDefault(windowSize) {
+   return windowSize !== null && windowSize !== void 0 ? windowSize : 21;
+ }
+-function findLastWhere(arr, predicate) {
+-  for (var i = arr.length - 1; i >= 0; i--) {
+-    if (predicate(arr[i])) {
+-      return arr[i];
+-    }
+-  }
+-  return null;
+-}
+ 
+ /**
+  * Base implementation for the more convenient [`<FlatList>`](https://reactnative.dev/docs/flatlist)
+@@ -1119,7 +1111,8 @@ class VirtualizedList extends StateSafePureComponent {
+       _keylessItemComponentName = '';
+       var spacerKey = this._getSpacerKey(!horizontal);
+       var renderRegions = this.state.renderMask.enumerateRegions();
+-      var lastSpacer = findLastWhere(renderRegions, r => r.isSpacer);
++      const lastRegion = renderRegions[renderRegions.length - 1];
++      const lastSpacer = lastRegion?.isSpacer ? lastRegion : null;
```

Comment:
Maybe we should actually use es5 syntax here just in case

```suggestion
+      var lastSpacer = lastRegion != null && lastRegion.isSpacer ? lastRegion : null;
```

Reply:
Thanks!